test/unit/rustdoc/rustdoc_test_dep_with_alias.rs

```diff
@@ -0,0 +1,6 @@
+/// ```
+/// let v = 42;
```

Comment:
Does the test still work if you use `aliased_adder` in the test? Feels like that'd be good regression testing to have too.

Reply:
No it will not, because tests also are not inherit `aliases` from crate. In test you still can workaround it because it has it's own `aliases` you can specify explicitly. I thought it's probably a topic for the separate PR when I was thinking about fixing it here :sweat_smile:

Reply:
Or maybe I misunderstood you question :sweat_smile: Did you mean if I use `aliased_adder` in doc?

Reply:
Yes! :)